archivers/afio/Portfile

```diff
@@ -1,43 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        kholtman afio 2.5.2 v
+revision            0
+checksums           rmd160  a0da5fafb6cee3fac054bbbb74eacc8e24cd7c7f \
+                    sha256  c64ca14109df547e25702c9f3a9ca877881cd4bf38dcbe90fbd09c8d294f42b9 \
+                    size    194030
 
-name                afio
-version             2.5.1
 categories          archivers
-platforms           darwin
 maintainers         {puffin.lb.shuttle.de:michael.klein @mklein-de}
```

Comment:
could you add openmaintainer and remove the puffin.lb.shuttle.de address while you're at it?

Reply:
Most of the other ports you maintain also list this email address. If you'd like to change it to a different address and/or add openmaintainer, may I request that you prepare and submit a new PR to do so in all your ports, after this PR is merged?

Are you ok with this PR otherwise? If so, I'll merge it.

Reply:
Yes, that's fine with me.